jest.config.js

```diff
@@ -0,0 +1,27 @@
+export default {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  roots: ["./src"],
+  silent: false,
+  verbose: true,
+  coveragePathIgnorePatterns: [
+    "<rootDir>/src/lesson4/__tests__",
```

Comment:
You can add the specific files you want to ignore in the coveragePathIgnorePatterns option.